include/swift/AST/Expr.h

```diff
@@ -323,11 +323,12 @@ class alignas(8) Expr : public ASTAllocated<Expr> {
     NumCaptures : 32
   );
 
-  SWIFT_INLINE_BITFIELD(ApplyExpr, Expr, 1+1+1+1+1+1+1,
+  SWIFT_INLINE_BITFIELD(ApplyExpr, Expr, 1+1+1+1+1+1+1+1,
```

Comment:
Whenever you make a change to these bitfields, I believe you might have to increment the `SWIFTMODULE_VERSION_MINOR` counter at the top of `ModuleFormat.h`. Though maybe we don't serialize Expr's? I'd have to check

Reply:
This isn't necessary, because we don't serialize `Expr` nodes.